pointer-masking-proposal.adoc

```diff
@@ -158,41 +155,47 @@ WPRI,UTE,WPRI,PM
 Figure 3d: Memory Tagging Extension register (_**umte**_) for RV128 (U-mode) +
  +
 
-[cols=",",]
+[cols="15%,15%,70%", options="header"]
 |===========================================================================================================================
-|*Value* |*Meaning*
-|00 |PM is disabled (_default_) and _**xPMMASK**_ and _**xPMBASE**_ registers can only be modified by the higher privilege level
-|01 |PM is enabled and _**xPMMASK**_ and _**xPMBASE**_ registers can only be modified by the higher privilege level
-|10 |PM is disabled and _**xPMMASK**_ and _**xPMBASE**_ registers can be modified by the same privilege level
-|11 |PM is enabled and _**xPMMASK**_ and _**xPMBASE**_ registers can be modified by the same privilege level
+^|*PM Bits* ^|*Name* ^|*Meaning*
+^.^| PM[0:0] ^.^| PM.Enabled |
+
+  0 – PM is disabled (_default_) +
+  1 – PM is enabled
+
+^.^| PM[1:1] ^.^| PM.Current |
+
+ 0 – _**xPMMASK**_ and _**xPMBASE**_ registers can only be modified by the higher privilege level +
```

Comment:
Could you please describe the behavior when access to xPMMASK/xPMBASE/MMTE is violated? I.e. should Illegal instrucion be raised?

Reply:
Yes, that's an interesting question. Together with Allen and Andy we have a separate mail-thread where Allen raised the same question.
 - "if (curr_mode< CSR_mode), a CSR  access to MMTE, Base or Mask will trap" (Allen's quote)
 - if PM.Current=0, any write to it or associated Mask/Base would simply be ignored. That's the current proposal (not reflected in that doc yet). However, if someone has different opinion and use-case we should re-discuss that.

Reply:
Actually, I think I prefer that the write be ignored. This makes it much more similar to a lock bit , and prevents violating trapping behavior being  determined by currmode and CSRaddr[9:8]